korrekt/src/test/integration_tests_axiom.rs

```diff
@@ -0,0 +1,547 @@
+#[cfg(test)]
+#[cfg(feature = "use_axiom_halo2_proofs")]
+mod tests {
+    use crate::circuit_analyzer::analyzer::Analyzer;
+    use crate::io::{
+        analyzer_io_type,
+        analyzer_io_type::{AnalyzerOutputStatus, VerificationInput, VerificationMethod},
+    };
+    use crate::sample_circuits::axiom as sample_circuits;
+    use halo2curves::bn256;
+    use num::{BigInt, Num};
+    use axiom_halo2_proofs::halo2curves::bn256::Fr;
+    use std::collections::HashMap;
+    use std::marker::PhantomData;
+
+    #[test]
+    fn create_analyzer_test() {
+        let circuit = sample_circuits::bit_decomposition::two_bit_decomp_axiom::TwoBitDecompCircuit::<
+            Fr,
+        >::default();
+        let k: u32 = 11;
+
+        let mut analyzer = Analyzer::new(&circuit, k).unwrap();
+
+        assert!(analyzer.cs.gates.len().eq(&3));
+        assert!(analyzer.cs.degree().eq(&3));
+        //assert!(analyzer.cs.num_advice_columns().eq(&3));
+        //assert!(analyzer.cs.num_instance_columns().eq(&1));
+        //assert!(analyzer.cs.num_selectors.eq(&1) || analyzer.cs.num_fixed_columns().eq(&1));
```

Comment:
These are commented out - can they be removed, or should there be a ticket to follow up on them?

Reply:
These were commented out while testing, they are valid assertions, so we can keeping them.

Reply:
ah, should they be uncommented then?